### PR TITLE
HIPCommander code cleanup

### PR DESCRIPTION
**Pull request by ansurya**
=>Removed dependency on HSA and HC. 
=>Corrected scripts to use latest hipCommander syntax.
=>updated code to pass args as per module API 

Issues Addressed:
https://github.com/ROCm-Developer-Tools/HIP/issues/808

### CODE REVIEW

File: samples/1_Utils/hipCommander/hipCommander.cpp [hunk 1]
```diff
@@ -9,8 +9,6 @@
 #include <hip/hip_runtime.h>
 #ifdef __HIP_PLATFORM_HCC__
 #include <elf.h>
```
Comment (mangupta) [inline samples/1_Utils/hipCommander/hipCommander.cpp]: I think this include is also not required. We should be able to remove it.

Comment (ansurya) [reply samples/1_Utils/hipCommander/hipCommander.cpp]: Fixed.

**Comment by gargrahul** [comment]
@mangupta Shall we keep AQLCommand test ? OR with hipLauncKernelGGL , there shouldn't be any difference ? I think earlier AQLCommand was useful since PFE was used.

**Comment by mangupta** [comment]
@gargrahul The reason why we are removing aqlcommand test is because of hip-clang and vdi. hipCommander sample should purely be using only HIP APIs. If we need aql tests, we should keep them separate from the samples.

File: samples/1_Utils/hipCommander/hipCommander.cpp [hunk 2]
```diff
@@ -8,9 +8,6 @@
 
 #include <hip/hip_runtime.h>
 #ifdef __HIP_PLATFORM_HCC__
```
Comment (mangupta) [inline samples/1_Utils/hipCommander/hipCommander.cpp]: I don't see the point of keeping an empty ```#ifdef ... #endif```.

**Comment by bensander** [comment]
the HSA/ROCR modes could be useful for latency comparisons.  We could make these an #ifdef so tool still compiles on systems where ROCR is not available.

**Pull request merged**